### ISSUE TITLE
Insufficient contrast for active command bar buttons in high contrast mode

### ISSUE THREAD

**Issue by pownkel** [microsoft/accessibility-insights-web#2736]
## Describe the bug

Currently in both web and electron, when we click one of the command bar buttons (for example, start over), the text becomes the default text color (normally black) and the icon becomes a very dark blue:

<img width="76" alt="button active" src="https://user-images.githubusercontent.com/55459788/82507963-87217c00-9ab8-11ea-91df-260cc064efe3.PNG">

In high contrast mode, the dark blue icon does not have sufficient contrast against the black background (should be 3:1):

<img width="80" alt="button active high contrast" src="https://user-images.githubusercontent.com/55459788/82508029-af10df80-9ab8-11ea-83a6-50c301119d03.PNG">

<img width="454" alt="contrast" src="https://user-images.githubusercontent.com/55459788/82508240-4b3ae680-9ab9-11ea-817f-7ce2395d17e8.PNG">

**To Reproduce**

1. Start an assessment with the web extension
2. Turn on high contrast mode
3. Click and hold down the start over or export report button (or set the state to active with devtools)

## Context (please complete the following information)

- OS Name & Version: Windows 10 Build 18363.836
- AI-Web Version & Environment: 2.18.2 prod
- Browser Version: Google Chrome 81.0.4044.138 (Official Build) (64-bit)
- Target Page: any page
